src/palace/manager/api/millenium_patron.py

```diff
@@ -105,7 +107,6 @@ def validate_neighborhood_mode(cls, v):
                 AuthenticationMode.FAMILY_NAME: "Family Name",
             },
         ),
-        alias="auth_mode",
```

Comment:
I removed a bunch of old unused aliases, that were left over from when we migrated from the settings table. These weren't used anymore, and it seemed like a good time to remove them.